src/next-collection-view.js

```diff
@@ -577,6 +577,17 @@ const CollectionView = Backbone.View.extend({
     this.Dom.appendContents(this.el, els, {_$el: this.$el});
   },
 
+  swapChildViews(view1, view2) {
+    if (!this.children.hasView(view1) || !this.children.hasView(view1)) {
```

Comment:
maybe so.. but I think we'd have to check the `parentNode` in this function so that the children don't also swap.  We'd also have to document clearly that only attached children can be swapped, but I think that probably makes sense.